### PR TITLE
Feat: Added Inter font to learn how font loading works, also Packpal logo and how to style Clerk buttons

### PR DESCRIPTION
**Pull request by acton-bell**
Fonts:
https://nextjs.org/docs/basic-features/font-optimization

One way (among many) to style Clerk buttons:
https://beta-docs.clerk.com/components/authentication/sign-in-button#custom-usage

### CODE REVIEW

File: public/packpal-transparentbg.png
Comment (josianed) [inline public/packpal-transparentbg.png:1]: 🐭

File: src/pages/index.tsx [hunk 1]
```diff
@@ -13,19 +17,42 @@ const Home: NextPage = () => {
...
         <meta name="description" content="Generated by create-t3-app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <main className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-[#2e026d] to-[#15162c]">
-        <div className=" bg-slate-200">
-          <p>User?.username: {JSON.stringify(user?.emailAddresses)}</p>
+      <main
+        className={`flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-[#55a490] to-[#c67744] ${inter.className}`}
+      >
+        <img
```
Comment (josianed) [inline src/pages/index.tsx:27]: you could use next Image here (tutorial example at 1:04:00 [here](https://www.youtube.com/watch?v=YkOSUVzOAA4) and docs [here](https://nextjs.org/docs/api-reference/next/image))

Comment (josianed) [reply src/pages/index.tsx:27]: unclear if the features of the Image components are worth using at this stage though so could always update to use it later on

File: src/pages/index.tsx [hunk 2]
```diff
@@ -13,19 +17,42 @@ const Home: NextPage = () => {
...
+          src="./packpal-transparentbg.png"
+          className="drop-shadow-lg"
+          alt="Packpal Logo"
+          width={250}
+          height={250}
+        />
+        <div className="w-1/2 rounded border bg-slate-200 p-4 shadow-lg">
+          <p>User?.username: {JSON.stringify(user?.fullName)}</p>
```
Comment (josianed) [inline src/pages/index.tsx:35]: nice. tried logging in with my personal google account and it showed my full name correctly

**Review by josianed — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (acton-bell) [reply src/pages/index.tsx:27]: It made the resolution a bit low and made Ermie look fuzzy :(

Comment (acton-bell) [reply src/pages/index.tsx:27]: But yes we should use it :)

**Pull request merged**